arkouda/series.py

```diff
@@ -193,13 +192,13 @@ def isin(self, lst):
         boolean = in1d(self.values, lst)
         return Series(data=boolean, index=self.index)
 
-    def locate(self, key):
+    @typechecked
+    def locate(self, key: Union[int, pdarray, "Series", List, Tuple]):
```

Comment:
here and in other places, we need the returns part of the typechecking
```python
def locate(self, key: Union[int, pdarray, Series, List, Tuple]) -> Series:
```

Reply:
Done